src/RemovePtrToInt.cc

```diff
@@ -80,24 +71,23 @@ visitIntToPtrInst(IntToPtrInst *I2P, Function &F, const DataLayout &DL,
   if (!PN)
     return false;
 
-  RPTI_LOG(dbgs() << F.getName() << ":\n");
-  RPTI_LOG(I2P->dump());
-  RPTI_LOG(PN->dump());
+  RPTI_LOG(outs() << F.getName() << ":\n"; I2P->print(outs());
```

Comment:
I prefer `errs()` instead of `outs()`...

Reply:
OK, will update in a minute. Why is errs better?

Reply:
Sometimes, it is convenient to run with logs enabled during batch runs. In these cases, we spilt stdout and stderr into different files. It is convenient when all debug-related messages are separated from the normal output.